src/iperf.h

```diff
@@ -430,9 +430,10 @@ struct iperf_test
 extern int gerror; /* error value from getaddrinfo(3), for use in internal error handling */
 
 /* UDP "connect" message and reply (textual value for Wireshark, etc. readability - legacy was numeric) */
-#define UDP_CONNECT_MSG 0x36373839          // "6789" - legacy value was 123456789
-#define UDP_CONNECT_REPLY 0x39383736        // "9876" - legacy value was 987654321
-#define LEGACY_UDP_CONNECT_REPLY 987654321  // Old servers may still reply with the legacy value
+#define UDP_CONNECT_MSG 0x36373839                 // "6789" - legacy value was 123456789
+#define UDP_CONNECT_REPLY 0x39383736               // "9876" - legacy value was 987654321
+#define LEGACY_UDP_CONNECT_REPLY 987654321         // Old servers may still reply with legacy value
+#define LEGACY_UDP_CONNECT_REPLY_ENDIAN 2976439866 // Old servers may still reply with legacy value
```

Comment:
Per issue #1608, reverse Endian of `UDP_CONNECT_REPLY` should also be considered. therefore should also add:
```c
#define UDP_CONNECT_REPLY_ENDIAN 0x36373839
```